grep-00XX-relicense-volk.md

```diff
@@ -0,0 +1,172 @@
+Especially, long term VOLK contributors expressed their desire to be able to use VOLK in a broader set of projects.
+
+### State of the GREP
+
+This GREP received a lot of feedback and incorporates this feedback.
+Now, we consider it ready for submission.
+
+
+## Copyright / License
+
+This GREP is licensed as CC-BY-ND.
+Copyright 2021 Johannes Demel and any co-author listed above.
+
+
+## Motivation
+
+We recognize the desire of our long term contributors to be able to use VOLK with their contributions in their projects.
+This may explicitly include proprietary projects.
+We want to enable all contributors to be able to use VOLK wherever they want.
+At the same time, we want to make sure that improvements to VOLK itself are easily obtained by everyone, i.e. strike a balance between permissiveness and strict copyleft.
+With a formal process and a clear vision laid out, we hope to be able to get everyone on board.
+
+Since VOLK is a library it should choose a fitting license.
+If we see greater adoption of VOLK in more projects, we are confident that we will receive more contributions.
+May it be bug fixes, new kernels or even contributions to core features.
+
+
+## Description
+
+In order to facilitate this relicensing effort, we need to got through several steps.
+
+
+### Notification
+
+We consider it to be very important to let every contributor know that the wish to relicense by some contributors exists.
+Based on the information we can gather through git commit names and email addresses as well as GitHub accounts, we notify every contributor of this GREP and ask them to resubmit their contributions.
+
+
+### The new license
+
+We propose [LGPLv3+](https://www.gnu.org/licenses/lgpl-3.0.html) for the new license.
+LGPLv3+ strikes a balance between permissiveness and obligation to provide modified source code.
+All modifications to VOLK must be made available under the same terms under which everyone can use VOLK.
+This should not require users to share code that uses VOLK unaltered.
+
+
+### All contributors need to re-submit their code under the new license
+
+Whether one submitted code under a CLA or with a DCO, every contributor must re-submit their code under the new license.
+A contributor may submit their code to any project, including VOLK under a new license, even if that code was submitted under a CLA.
+
+
+### The process
+
+Currently, we have around 65 contributors to VOLK.
+In order to relicense VOLK, every contributor must re-submit their code under the new license.
+During the transition period, we want to be able to accept new contributions.
+
+1. We notify every contributor that we want to relicense VOLK.
+2. New contributions are required to be submitted under the new license.
+3. Contributors are kindly asked to resubmit their contributions.
+4. After all contributors have resubmitted their contributions, we have a new major release and continue VOLK development under the new license.
```

Comment:
After all, or after a vast/useful majority?

Reply:
Note: You can refer to the section below which talk about what happens if we have non-signers.

Reply:
I would say here that this "4" is the ideal, but reality is that we will keep moving forward (2) regardless of the state of the relicensing effort. I'd advise restating this item.